src/admin/fusionauth/fusionauth.service.ts

```diff
@@ -434,4 +433,84 @@ export class FusionauthService {
       }
     }
   }
+
+  async createUser(user: UserRequest): Promise<{userId: UUID, user: User, err: Error}> {
+    return this.fusionauthClient
+      .createUser(null, user)
+      .then(
+        (
+          response: ClientResponse<UserResponse>,
+        ): { userId: UUID; user: User, err: Error } => {
+          console.log('Found user');
+          return {
+            userId: response.response.user.id,
+            user: response.response.user,
+            err: null,
+          };
+        },
+      )
+      .catch((e): { userId: UUID; user: User, err: Error } => {
+        console.log(`Could not create user ${user}`, JSON.stringify(e));
+        return {
+          userId: null,
+          user: null,
+          err: e
+        };
+      });
+  }
+
+  async updateUser(userId: string, user: UserRequest): Promise<{_userId: UUID, user: User, err: Error}> {
+    return this.fusionauthClient
+      .patchUser(userId, user)
+      .then(
+        (
+          response: ClientResponse<UserResponse>,
+        ): { _userId: UUID; user: User; err: Error } => {
+          console.log('Found user');
+          return {
+            _userId: response.response.user.id,
+            user: response.response.user,
+            err: null
+          };
+        },
+      )
+      .catch((e): { _userId: UUID; user: User; err: Error } => {
+        console.log(`Could not update user ${user.user.id}`, JSON.stringify(e));
+        return {
+          _userId: null,
+          user: null,
+          err: e
+        };
+      });
+  }
+
+  async changePassword(data: {loginId: string, password: string}): Promise<any> {
```

Comment:
`updatePassword` already exists. Try to overload that implementation.

Reply:
Theres difference between those...one requires users old password for resetting, the newer one added just force updates the password for a user on the baqsis of loginId